csrc/alias_analysis.cpp

```diff
@@ -426,15 +426,26 @@ bool okToRelayout(
                                             : tv->getMaybeAllocationDomain());
   return new_layout.isCompliantWith({allocation, tv->getContiguity()});
 }
+
+bool isGlobalTensor(const TensorView* tv) {
+  return tv->isFusionInput() || tv->isFusionOutput() ||
+      std::any_of(tv->uses().begin(), tv->uses().end(), [](Expr* expr) {
+           // This is based on that Resize op triggers a segmentation in
+           // existing schedulers.
+           return expr->isA<PadOp>() || expr->isA<SliceOp>() ||
```

Comment:
I'm seeing that in the example I have, `SliceOp` doesn't always give us segment, when it's a static shape with a no-op slice.
NOTE for myself. that's the logic that we might want to mimic here.